packages/did-method-element/README.md

```diff
@@ -2,114 +2,220 @@
 
 This package contains an implementation of Sidetree Core, using Ethereum and IPFS
 
-## Usage
+## Run Tests
+
+This package provides a series of tests, which act as a reference for the functionality for Element.
+The tests can be run by cloning this repository with the following commands.
 
 ```
-npm install --save @sidetree/element
+$ git clone https://github.com/transmute-industries/sidetree.js.git
+$ cd sidetree.js/packages/did-method-element
+$ npm install
+$ npm run test
 ```
 
-## Development
+## Set Up a Node
+
+Instructions for setting up an Element node are defined in the following documents.
 
 - [Install Local Development with Ganache](docs/local-dev.md)
 - [Install Testnet Development with Ropsten](docs/local-element-ropsten-install.md)
 
-## Run Tests
+## As a Library
+
+This package is published to NPM and can be included in your own projects with
 
 ```
-npm install
-npm run test
+npm install @sidetree/element --save
 ```
 
 ## Element DID Method Specification
 
 Element is an implementation of the Sidetree Protocol that uses the Ethereum blockchain as the ledger layer and IPFS as the content-addressable storage layer
-
 For more information, see [the sidetree spec](https://identity.foundation/sidetree/spec/)
 
 ## Method syntax
 
 The namestring identifying this did method is `elem`
-
 A DID that uses this method MUST begin with the following prefix: `did:elem`. Per the DID specification, this string MUST be in lowercase.
 
 An additional optional network specific identifier may be added as such
+
 - `did:elem:ropsten:EiBOWH8368BI9NSaVZTmtxuqwpfN9NwAwy4Z95_VCl6A9g`
 - `did:elem:mainnet:EiBOWH8368BI9NSaVZTmtxuqwpfN9NwAwy4Z95_VCl6A9g`
 - `did:elem:EiBOWH8368BI9NSaVZTmtxuqwpfN9NwAwy4Z95_VCl6A9g`
 
-By default, if the network specific identifier is not present, then the default is `ropsten`.
-The default may change in the future once `mainnet` is supported.
-
-The remainder of a DID after the prefix, called the did unique suffix, MUST be `SHA256` hash of the encoded create payload (see below)
-
-## Format and Encoding
-
-Base64URL encoded Multihashes are used.
+By default, if the network specific identifier is not present, then the default is `mainnet`.
```

Comment:
I changed the default identifier from Ropsten to Mainnet. I don't think it's a good idea to chance references.